drenso/symfony-shared/0.6/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+    "bundles": {
+        "Drenso\\Shared\\DrensoSharedBundle": ["all"]
+    },
+    "copy-from-recipe": {
```

Comment:
The `copy-from-package` might be more flexible, as the configuration will still change quite a lot in the coming time.